index.js

```diff
@@ -22,6 +22,8 @@ class LogForwardingPlugin {
       this.serverless.service.resources = {
         Resources: {},
       };
+    } else if (this.serverless.service.resources.Resources === undefined) {
+      this.serverless.service.resources.Resources = {};
```

Comment:
`updates the resources object if it doesn\'t exist` test was failing due to `this.serverless.service.resources` being instantiated by Serverless framework but didn't have `Resources` property set.